### ISSUE TITLE
(WHM / AST): Replace unused lower level abilities with their higher level counterpart when available

### ISSUE THREAD

**Issue by ceci-woodward** [attickdoor/XIVComboPlugin#88]
### Scenario

Upon queueing for roulettes / lower-level dungeons as a healer often times you have to swap out Cure II / Benefic II with their lower-level counterparts.

This can be a nuisance and in my case I usually have it stashed into the corner of one of my bars so I can quickly swap when I need to.

### Proposed Solution

In my opinion, this feels like the kind of QoL feature that would fit perfectly into this plugin!

Basically adding an option for White Mage and Astro that swaps out their respective abilities based upon level.

**Comment by DenL** [comment]
The feature requested is to replace Cure 2 with Cure 1 when Cure 2 isn't available due to being synced down. It in no way affect those who still want to use both spells, even with the feature turned on.
